<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QloT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../images/qlot_512.svg</normaloff>../../images/qlot_512.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion de stock</string>
     </attribute>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>721</width>
        <height>451</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_critere">
        <property name="text">
         <string>Critère de recherche</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="critere_combo_box_gestion">
        <item>
         <property name="text">
          <string>Modele</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Couleur</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Taille</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quantité</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date de livraison</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prix d'achat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prix de vente</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rabais</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="ligne_recherche_gestion"/>
      </widget>
      <widget class="QTableWidget" name="tableau_recherche_gestion">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>8</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>140</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Modèle</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Couleur</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Taille</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Quantité</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date de livraison</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prix d'achat</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prix de vente</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rabais</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Vente</string>
     </attribute>
     <widget class="QSplitter" name="splitter_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>721</width>
        <height>451</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_critere_3">
        <property name="text">
         <string>Critère de recherche</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="critere_combo_box_vente">
        <item>
         <property name="text">
          <string>Identifiant</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Articles vendus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prix total</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="ligne_recherche_vente"/>
      </widget>
      <widget class="QTableWidget" name="tableWidget_vente">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>180</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Articles vendus</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prix total</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvelle_BDD"/>
    <addaction name="actionOuvrir_une_BDD"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionVersion"/>
    <addaction name="actionA_propos_de_QloT"/>
   </widget>
   <widget class="QMenu" name="menuProduits">
    <property name="title">
     <string>Produits</string>
    </property>
    <addaction name="actionAjouter_une_marchandise"/>
    <addaction name="actionAjouter_une_vente"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuProduits"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouvelle_BDD">
   <property name="text">
    <string>Nouvelle BDD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOuvrir_une_BDD">
   <property name="text">
    <string>Ouvrir une BDD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionA_propos_de_QloT">
   <property name="text">
    <string>A propos de QloT ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAjouter_une_marchandise">
   <property name="text">
    <string>Ajouter une marchandise</string>
   </property>
  </action>
  <action name="actionAjouter_une_vente">
   <property name="text">
    <string>Ajouter une vente</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
